rotkehlchen/api/rest.py

```diff
@@ -2850,6 +2850,22 @@ def get_asset_icon(
             )
         return maybe_create_image_response(image_path=image_path)
 
+    def get_protocol_icon(
+            self,
+            protocol: str,
+            match_header: Optional[str],
+    ) -> Response:
+        root_dir = Path(__file__).resolve().parent.parent.parent
+        icon_path = root_dir / 'frontend' / 'app' / 'public' / 'assets' / 'images' / 'protocols' / AIRDROPS.get(protocol)[4]  # noqa: E501
+        response = check_if_image_is_cached(
+            image_path=icon_path,
+            match_header=match_header,
+        )
+        if response is not None:
+            return response
+        else:
+            return create_image_response(image_path=icon_path)
```

Comment:
There is no need to send the file in a response. What the frontend needs is the file name so they can get it. This can go

Reply:
see, it was a part of the requirements here:

https://github.com/rotki/rotki/issues/2002

he specified needing a new endpoint for the icons as a result